grunt/shell.js

```diff
@@ -1,14 +1,17 @@
 module.exports = {
+  ulimit: {
+    command: 'ulimit -n 9999'
```

Comment:
This won't do what you want. The ulimit command is used to set limit for the current shell and any spawned processes. The grunt-shell plugin spawns a new shell to run its tasks so this command will only affect commands in the same shell not other grunt tasks.

To resolve the issue i would remove this task and add the ulimit statement to the top of the problem bash script at `docs/script/build.sh` fixing it at the source.